docs/guide/dash-features-masternode-payment.md

```diff
@@ -4,6 +4,6 @@ Since [DIP3](https://github.com/dashpay/dips/blob/master/dip-0003.md) (introduce
 
 On-chain masternode lists reduce the complexity of reward payments, make payments much more predictable, and also allow masternode payments to be enforced for all blocks (enforcement for superblocks was not possible in the previous system).
 
-## High-Performance Masternodes
+## Evolution Masternodes
 
-Since high-performance masternodes (HPMNs) have four times more collateral than regular masternodes, HPMNs will be paid in four consecutive blocks each time they are selected for payment to maintain the same incentive structure that regular masternodes have. Additionally, HPMNs will receive 100% of the fees generated from Dash Platform since they are the only nodes that host it.
+Since evolution masternodes (EvoNodes) have four times more collateral than regular masternodes, EvoNodes will be paid in four consecutive blocks each time they are selected for payment to maintain the same incentive structure that regular masternodes have. Additionally, EvoNodes will receive 100% of the fees generated from Dash Platform since they are the only nodes that host it.
```

Comment:
```suggestion
Since Evolution masternodes (EvoNodes) have four times more collateral than regular masternodes, EvoNodes will be paid in four consecutive blocks each time they are selected for payment to maintain the same incentive structure that regular masternodes have. Additionally, EvoNodes will receive 100% of the fees generated from Dash Platform since they are the only nodes that host it.
```